Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -5,17 +5,21 @@ import XCTest
 final class MongoDatabaseTests: MongoSwiftTestCase {
     static var allTests: [(String, (MongoDatabaseTests) -> () throws -> Void)] {
         return [
-            ("testDatabase", testDatabase)
+            ("testMongoDatabase", testMongoDatabase)
         ]
     }
 
+    override internal class var testDatabase: String {
```

Comment:
do we have a reason to override this, or should we just use the default name?

I have the same question for all the other classes where we are doing this. if the overriding is just to be consistent with the names I originally chose, we don't need to stick to them.

Reply:
Yeah, the only reason any overrides are in here is just to preserve what database was used before. I can make them all use the default.

Reply:
Done